cli/command/container/opts.go

```diff
@@ -683,6 +683,31 @@ func parse(flags *pflag.FlagSet, copts *containerOptions, serverOS string) (*con
 	}, nil
 }
 
+func parseDefaultNetOpts(copts *containerOptions) (map[string]*networktypes.EndpointSettings, error) {
+	if copts.aliases.Len() > 0 {
+		return nil, errors.New("network-scoped aliases are only supported for user-defined networks")
+	}
+	NetworkMode := container.NetworkMode(copts.netMode.NetworkMode())
+	EndpointsConfig := make(map[string]*networktypes.EndpointSettings)
+	if copts.ipv4Address != "" || copts.ipv6Address != "" || copts.linkLocalIPs.Len() > 0 {
+		epConfig := &networktypes.EndpointSettings{}
+		EndpointsConfig[string(NetworkMode)] = epConfig
```

Comment:
nit: local variables shouldn't start with upper case. It's weird the linter didn't catch that 🤔

Reply:
Yes, I have modified it